### PR TITLE
[SEASK-17] Quick Start -> Data Quality

### PR DESCRIPTION
**Pull request by semalaiappan**

### CODE REVIEW

File: sample-api-calls/Data Quality/DistructErrorStaging.sql [hunk 1]
```diff
@@ -0,0 +1,25 @@
+
+USE EdFi_Ods_2022
+GO
+
```
Comment (tmeador-dl) [inline sample-api-calls/Data Quality/DistructErrorStaging.sql]: are we ok with hardcoding the database names? we would have to remember to update if we change the name/year

File: sample-api-calls/Data Quality/DistructErrorStaging.sql [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+/****** Object:  Table [dbo].[DeployJournal]    Script Date: 5/17/2021 11:54:45 AM ******/
```
Comment (tmeador-dl) [inline sample-api-calls/Data Quality/DistructErrorStaging.sql]: this can probably be removed

**Review by tmeador-dl — CHANGES_REQUESTED** [review]
can we update the sql scripts to use either tabs or spaces

File: sample-api-calls/Data Quality/DistructErrorStaging.sql (hunk 1 above)
Comment (semalaiappan) [reply sample-api-calls/Data Quality/DistructErrorStaging.sql]: I removed it . Thanks

File: sample-api-calls/Data Quality/DistructErrorStaging.sql (hunk 2 above)
Comment (semalaiappan) [reply sample-api-calls/Data Quality/DistructErrorStaging.sql]: I removed it . Thanks

**Review by tmeador-dl — APPROVED** [review]

**Pull request merged**